presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -596,6 +596,40 @@ private void testCreatePartitionedBucketedTableAsFewRows(Session session, HiveSt
         assertFalse(getQueryRunner().tableExists(session, tableName));
     }
 
+    @Test
+    public void testCreateBucketedSortedTable()
```

Comment:
This test seems redundant with the one in `testShowCreateTable` and doesn't actually validate that we created a sorted table. Probably just remove it.